Wikipedia/Code/TabsOverviewCoordinator.swift

```diff
@@ -34,6 +34,56 @@ final class TabsOverviewCoordinator: Coordinator {
         self.dataController = dataController
     }
     
+    private func surveyViewController() -> UIViewController {
+        
+        var wikimediaProject: WikimediaProject? = nil
+        if let siteURL = dataStore.languageLinkController.appLanguage?.siteURL,
+        let project = WikimediaProject(siteURL: siteURL) {
+            wikimediaProject = project
+        }
+        
+        let subtitle = WMFLocalizedString("tabs-survey-title", value: "Help improve the tabs feature. Are you satisfied with this feature?", comment: "Title for article tabs survey")
+        
+        let surveyLocalizedStrings = WMFSurveyViewModel.LocalizedStrings(
+            title: CommonStrings.activityTabSurveyTitle,
```

Comment:
Minor nitpick, but let's rename this CommonStrings var if we're going to reuse it across features. I think we're stuck with the localized string key or else we'll lose translations, but at least this name could feel more generic.